docs/04-troubleshooting/04-ssl-certs.md

```diff
@@ -0,0 +1,26 @@
+# GRID services and SSL
+
+GRID services make HTTPS calls between themselves, and need a root certificate to do so.
+
+## Generating certificate.
+
+Installation is trivial:
+
+Ensure JAVA_HOME is exported, and then run
+```
+mkcert -install
```

Comment:
Is this not done as part of `setup.sh`?

Reply:
This step is performed by [`dev-nginx`](https://github.com/guardian/dev-nginx/blob/e41c9afcea2c30f2d9c92b980c6f3c1ecaa5531d/script/setup-cert#L48), which `setup.sh` calls out to; I don't think this is needed here.

Reply:
Will move to a rectification section below.